.changelog/1657.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
This would probably be `improvement` instead of `feature` since we introduced the status reporting as a feature in the last release.

Reply:
Thanks. Amended 👍